satellite/console/consoleweb/utils.go

```diff
@@ -14,6 +15,18 @@ import (
 	"storj.io/storj/satellite/console/consoleweb/consoleql"
 )
 
+func init() {
+	err := mime.AddExtensionType(".ttf", "font/ttf")
+	if err != nil {
+		panic(err.Error())
```

Comment:
Should we just terminate the web process and log the error instead of panic here?

Reply:
we don't have access to logger here